ontotext-yasgui-web-component/src/models/ontotext-yasgui.ts

```diff
@@ -65,10 +66,23 @@ export class OntotextYasgui {
     return this.yasgui.getTab().getYasqe().query();
   }
 
+  getRanRequestsCount(): RanRequestCountInfo {
+    return this.getInstance().getRanRequestsCount();
+  }
+
   abortQuery(): void {
     this.yasgui.getTab().getYasqe().abortQuery();
   }
 
+  abortAllQuery(): void {
```

Comment:
`abortAllQueries` or `abortAllSelectQueries`? Which brings the question, why we have such method `abortAllQuery` at all?!